package.json

```diff
@@ -15,7 +15,7 @@
     "codecov": "^3.1.0",
     "mocha": "^5.2.0",
     "nyc": "^13.1.0",
-    "request": "^2.46.0"
+    "request": "2.46.0"
```

Comment:
According to [Snyk](https://app.snyk.io/org/pierluigi/test/github/a9a0d683-541c-404c-ae02-11854d2e6d6a/master..84435e1a170c37665daa85e258bb04949fc95d81?fromStatus=true&severityThreshold=high) this version contains a vulnerability. Let's keep the ^.

```suggestion
    "request": "^2.46.0"
```

Reply:
You're awesome!